memote/support/consistency.py

```diff
@@ -548,6 +548,37 @@ def find_metabolites_consumed_with_closed_bounds(model):
     return mets_consumed
 
 
+def find_metabolite_production_infeasibility(model):
```

Comment:
Did you have a look both at `find_metabolites_produced_with_closed_bounds` and `find_metabolites_consumed_with_closed_bounds`? They call `helpers.run_fba` to run the optimization. It makes sense to use modularity wherever it exists to avoid bugs and to capitalize on existing unit tests. In fact, I think you may have saved yourself some time copy/pasting the code from there, modifying it slightly afterwards.